### ISSUE TITLE
Getting error ERR_MODULE_NOT_FOUND 

### ISSUE THREAD

**Issue by andres-leon** [forsti0506/a11y-sitechecker#565]
I am getting the same error Here is the complete message I get:

```
node:internal/modules/esm/resolve:265
throw new ERR_MODULE_NOT_FOUND(
^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker_code' imported from /usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker.js
at finalizeResolution (node:internal/modules/esm/resolve:265:11)
at moduleResolve (node:internal/modules/esm/resolve:933:10)
at defaultResolve (node:internal/modules/esm/resolve:1157:11)
at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:383:12)
at ModuleLoader.resolve (node:internal/modules/esm/loader:352:25)
at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:227:38)
at ModuleWrap. (node:internal/modules/esm/module_job:87:39)
at link (node:internal/modules/esm/module_job:86:36) {
code: 'ERR_MODULE_NOT_FOUND',
url: 'file:///usr/local/lib/node_modules/a11y-sitechecker/bin/a11y-sitechecker_code'
}

Node.js v20.14.0

```

Installing on Ubuntu 22.04.4 LTS.

Thank you for any assistance you may be able to provide!

**Comment by jlk4p** [comment]
I am seeing this same problem with version 6.1.0 on macOS 14.7. 
I've tried running this as a script command.
It generates the ERR_MODULE_NOT FOUND error whether I'm running Node 20.17.0 (with npm 10.8.2) or Node 22.9.0 (with npm 10.8.3)
